src/foam/nanos/boot/NSpec.js

```diff
@@ -11,6 +11,17 @@ foam.CLASS({
     'foam.nanos.auth.EnabledAware'
   ],
 
+  constants: [
+    {
+      name: 'NSPEC_NAME_CTX_KEY',
```

Comment:
```suggestion
      name: 'NSPEC_CTX_KEY',
```
Since you are passing the entire NSPEC.